Source/DataTypes/SvgUnitCollection.cs

```diff
@@ -77,45 +80,43 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
         {
             if (value is string)
             {
-                var s = ((string)value).Trim();
-
-                if (s.Equals("inherit", StringComparison.OrdinalIgnoreCase))
-                    return null;
-
+                var points = ((string)value).Trim().Split(new char[] { ',', ' ', '\r', '\n', '\t' }, StringSplitOptions.RemoveEmptyEntries);
                 var units = new SvgUnitCollection();
-                units.IsNull = s.Equals("none", StringComparison.OrdinalIgnoreCase);
-                if (!units.IsNull)
-                    foreach (var point in s.Split(new char[] { ',', ' ', '\r', '\n', '\t' }, StringSplitOptions.RemoveEmptyEntries))
-                    {
-                        var newUnit = (SvgUnit)_unitConverter.ConvertFrom(point.Trim());
-                        if (!newUnit.IsNone)
-                            units.Add(newUnit);
-                    }
+
+                foreach (var point in points)
+                {
+                    var newUnit = (SvgUnit)_unitConverter.ConvertFrom(point.Trim());
+                    if (!newUnit.IsNone)
+                        units.Add(newUnit);
+                }
 
                 return units;
             }
 
             return base.ConvertFrom(context, culture, value);
         }
+    }
 
-        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)

```

Comment:
If destinationType is string, base class CanConvertTo always returns true and ConvertTo calls ToString.